components/host-dashboard/reports-section/TotalMoneyManagedSection.js

```diff
@@ -37,7 +51,102 @@ const TotalFundsLabel = styled(Container)`
   opacity: 80%;
 `;
 
-const TotalMoneyManagedSection = ({ currency, hostMetrics }) => {
+const ChartWrapper = styled.div`
+  position: relative;
+
+  .apexcharts-legend-series {
+    background: white;
+    padding: 8px;
+    border-radius: 10px;
+    & > span {
+      vertical-align: middle;
+    }
+  }
+
+  .apexcharts-legend-marker {
+    margin-right: 8px;
+  }
+`;
```

Comment:
If we're going to reuse the same styles for all charts, this should be exported to a common component. We can revisit later, but as a starter having this in `components/ChartWrapper.js` with a description like this could be a good start:
> Wraps a chart from ApexCharts to plug our custom styles to it

Reply:
Thanks, I've extracted this to a separate component.